processing/src/main/java/io/druid/query/BaseQuery.java

```diff
@@ -206,7 +206,7 @@ public boolean getContextBoolean(String key, boolean defaultValue)
     return parseBoolean(this, key, defaultValue);
   }
 
-  protected Map<String, Object> computeOverridenContext(Map<String, Object> overrides)
+  public Map<String, Object> computeOverridenContext(Map<String, Object> overrides)
```

Comment:
Changed this back to protected, adjusted call site to use `withOverriddenContext()`